### PR TITLE
KubeCon EU 2023 blog post

### PR DESCRIPTION
**Pull request by bailey**
## Hey, I just made a Pull Request!

Adding KubeCon EU 2023 blog post 🎉 

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: microsite/blog/2023-04-26-kubecon-eu-2023.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
...tage has officially hit over 1,000 adopters and 1,000 contributors – so it’s apt timing to modernize the governance model for the project. Taking pointers from the [CNCF Contributor Ladder Governance Template](https://contribute.cncf.io/maintainers/templates/), a new [Backstage Governance Model](https://github.com/backstage/backstage/blob/master/GOVERNANCE.md) is now in effect! Patrik walked us through the new ladder model which introduces a number of changes, one being the addition of [project area maintainers](https://github.com/backstage/backstage/blob/master/GOVERNANCE.md#project-area-maintainer). This role lets members of the community take increased ownership over a specific area of interest, like Catalog, Discoverability, TechDocs, Helm Charts, and Kubernetes. New project areas that will be added include Permissions and Software Templates. New project areas can be proposed by nominating a project area maintainer for the area. The new model also adds an organization member role for contributors who want to take a more active role in the Backstage community. You can open an issue to become an organization member [here](https://github.com/backstage/community/issues/new/choose).
```
Comment (Rugvip) [inline microsite/blog/2023-04-26-kubecon-eu-2023.md:22]:
```suggestion
Core maintainers [Ben Lambert](https://github.com/benjdlambert) and [Patrik Oldsberg](https://github.com/Rugvip) took center stage on Wednesday for the Backstage Maintainer Track: State of Backstage in 2023 talk. Backstage has officially hit over 1,000 adopters and 1,000 contributors – so it’s apt timing to modernize the governance model for the project. Taking pointers from the [CNCF Contributor Ladder Governance Template](https://contribute.cncf.io/maintainers/templates/), a new [Backstage Governance Model](https://github.com/backstage/backstage/blob/master/GOVERNANCE.md) is now in effect! Patrik walked us through the new ladder model which introduces a number of changes, one being the addition of [project area maintainers](https://github.com/backstage/backstage/blob/master/GOVERNANCE.md#project-area-maintainer). This role lets members of the community take increased ownership over a specific area of interest, like Catalog, Discoverability, TechDocs, Helm Charts, and Kubernetes. New project areas that will be added include Permissions and Software Templates. New project areas can be proposed by nominating a project area maintainer for the area. The new model also adds an organization member role for contributors who want to take a more active role in the Backstage community. You can open an issue to become an organization member [here](https://github.com/backstage/community/issues/new/choose).
```

File: microsite/blog/2023-04-26-kubecon-eu-2023.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title: 'ICYMI: KubeCon EU 2023'
+author: Bailey Brooks, Spotify
+authorURL: http://github.com/bailey
+authorImageURL: https://pbs.twimg.com/profile_images/1477424303192694785/qCfN6XWW_400x400.jpg
+---
+
+![KubeCon CloudNativeCon Europe 2023 banner](assets/2023-04-26/Backstage_News_3-7-23_KubeCon.png)
+
+We had a blast talking all things Backstage at KubeCon EU 2023 in Amsterdam last week. In case you missed it, here’s a quick recap:
+
```
Comment (Rugvip) [inline microsite/blog/2023-04-26-kubecon-eu-2023.md:11]: You'll want to add a truncate comment here for the fold, see other blog posts. Cba looking up the exact format as I'm on mobile :sweat_smile:

**Review by Rugvip — COMMENTED** [review]
Nice! :tada: 

Fairly certain the microsite build failure is unrelated, although I haven't seen it before. Think it's fine to ship @jhaals?

**Comment by jhaals** [comment]
Just prettier left

**Review by benjdlambert — APPROVED** [review]

**Comment by bailey** [comment]
Seeing `{/* truncate */}` in plain text on the blog post - should I remove? cc @Rugvip 
image

**Comment by benjdlambert** [comment]
@bailey it should be fixed now - was a short lived bug :joy:

**Pull request merged**